### ISSUE TITLE
opa-istio-plugin does not respect logging configuration

### ISSUE THREAD

**Issue by danielpops** [open-policy-agent/opa#1580]
# Expected Behavior
The main `--log-format` parameter passed to opa should be respected by plugins

# Actual Behavior
The `opa-istio-plugin` logs with the default log format regardless of the value of `--format`

# Steps to Reproduce the Problem

Just run the plugin container with `--log-format json` parameter and observe that the main opa server logs are in json, but the grpc server are in the default format.
```
$ docker run --rm -it opa-istio
--plugin-dir=/opt/opa/plugins run --server --addr=:8383 --log-level=debug --config-file=/opa/conf
ig/config.yaml --watch /opa/policies --log-format json
{"addrs":[":8383"],"insecure_addr":"","level":"info","msg":"Initializing server.","time":"2019-07-23T07:59:28Z"}
{"level":"info","msg":"Starting decision logger.","plugin":"decision_logs","time":"2019-07-23T07:59:28Z"}
INFO[0000] Starting gRPC server.                         addr=":9393" dry-run=true query=data.acceptance
```

**Comment by tsandall** [comment]
@danielpops is OPA loading the opa-istio plugin via an .so file? If so, I think this is due to known issues with the .so/vendoring setup. The plan is to remove the .so file and provide clear steps for how to customize OPA at build time instead of deploy time. /cc @patrick-east @ashutosh-narkar

**Comment by danielpops** [comment]
> @danielpops is OPA loading the opa-istio plugin via an .so file?

That's correct. 

Great to hear that you are addressing this (and other?) .so file loading issues. Happy to help test things out or brainstorm on things. I don't have any affinity to loading from .so files or anything, it's just all that I think is possible today

**Comment by ashutosh-narkar** [comment]
See comments in https://github.com/open-policy-agent/opa-istio-plugin/pull/131

**Comment by tsandall** [comment]
This looks like it's fixed in the latest version of opa-istio (0.14.1-istio-2). Closing for now. @danielpops if you see this again, please re-open this issue.
